containers/podman-desktop
# blog: add blog post about 5 things to know when user is coming from docker

### What does this PR do?

Add a blog post about 5 things to know when coming to Podman Desktop as a Docker user

### Screenshot/screencast of this PR

![image](https://user-images.githubusercontent.com/436777/226555491-4bb75701-e9ed-4951-aeed-d12f1999a9dd.png)


### What issues does this PR fix or reference?

fixes https://github.com/containers/podman-desktop-internal/issues/41


### How to test this PR?

Read the content from the netlify PR check link deploy of the website

Change-Id: If9f0e757298c435d91985e574543c8f402f17f31
